testnet/tools/icos_deploy.sh

```diff
@@ -369,8 +369,8 @@ rm -rf "${TMPDIR}"
 echo "-------------------------------------------------------------------------------"
 cd "${REPO_ROOT}/testnet/ansible"
 
-echo "**** Remove eventual monitoring - ($(dateFromEpoch "$(date '+%s')"))"
-ansible ic_p8s_service_discovery_destroy.yml
+# echo "**** Remove eventual monitoring - ($(dateFromEpoch "$(date '+%s')"))"
+# ansible ic_p8s_service_discovery_destroy.yml
```

Comment:
I suppose this is a temporary (dev) change, not to be merged?

Reply:
It was by accident. I removed it.